tests/requirements-linting.txt

```diff
@@ -8,6 +8,5 @@ mypy==0.971
 pre-commit==2.20.0
 pycodestyle==2.11.1
 pyflakes==3.1.0
-pyright==1.1.389
```

Comment:
Why not?

Reply:
See https://github.com/pydantic/pydantic/pull/11028#issuecomment-2512655920

Reply:
But yes both approaches are equivalent